tests_requre/service/test_views.py

```diff
@@ -1,17 +1,11 @@
 from flask import url_for
```

Comment:
would be nice to add some cases here, not just drop them

Reply:
done